packages/chains/src/moonbeam/assets.ts

```diff
@@ -243,6 +253,22 @@ export const assets: SupportedAsset[] = [
     oracle: OracleTypes.UniswapTwapPriceOracleV2,
     extraDocs: defaultDocs("https://moonbeam.moonscan.io", STELLA),
   },
+  {
+    symbol: assetSymbols.CELR,
+    underlying: CELR,
+    name: "CELR Token",
+    decimals: 18,
+    oracle: OracleTypes.UniswapTwapPriceOracleV2,
+    extraDocs: defaultDocs("https://moonbeam.moonscan.io", CELR),
+  },
+  {
+    symbol: assetSymbols.celrUSDT,
```

Comment:
This should use chainlink oracle for USDT

Reply:
this is not usdt. it's celr usdt.